src/coll_score/ucc_coll_score.h

```diff
@@ -19,8 +19,16 @@
 
 #define UCC_MSG_MAX UINT64_MAX
 
+typedef struct ucc_msg_range_fb {
+    ucc_list_link_t          list_elem;
+    ucc_score_t              score;
+    ucc_base_coll_init_fn_t  init;
+    ucc_base_team_t         *team;
+} ucc_msg_range_fb_t;
```

Comment:
rename this struct to generic "ucc_coll_entry" and unify this with "ucc_msg_range_t" struct with common members. this is not really fallback specific. for fallback colls you already have "ucc_list_link_t         fallback" in ucc_msg_range_t